rpms/README.md

```diff
@@ -5,53 +5,74 @@ needed to build a set of RPMs that are useful for working with Geotrellis.  If
 you are not intending to alter the composition or versions of the RPM
 packages, then you should be able to use the published binaries described in
 the README at the root of this project and do not need to build these
-artifacts yourself. Specifically:
+artifacts yourself.
 
-  > **Do nothing** if you are intending to deploy to EMR and have access to
-  > pre-built RPMs.  Simply point the EMR scripts to the relevant resources on
-  > S3.
+Built RPMs are publicly available at:
 
-  > **Fetch, do not build** if you have access to an already-built set of
-  > RPMs, and your purpose is to use these RPMs to build a Docker container.
+```
+s3://geotrellis-build-artifacts/rpms/
+```
 
-If you have a special use case, and the pre-built binaries are not sufficient,
-then you may need to build them yourself.
+If you need to load them during an EMR bootstrap, you can use `s3://geotrellis-build-artifacts/rpms/bootstrap.sh`.
 
-# Using this Repository #
+For example:
 
-## Fetching ##
+```
+BootstrapAction(
+  "Install GDAL 2.4 dependencies",
+  "s3://geotrellis-build-artifacts/rpms/bootstrap.sh",
+  "s3://geotrellis-build-artifacts", "gdal-2.4.1"
+)
+```
+
+## Caveats
+
+### GDAL S3 Credentials in EMR jobs
+
+Note that if you're installing a version of GDAL < 2.4.0 you'll need to restrict yourself to EC2 instance types that use the older Xen hypervisor. [A fix was added in GDAL 2.4.0](https://github.com/OSGeo/gdal/commit/9df23a3f09e5171e0051748c3de40151671cfea8#diff-6fe1009dc8083259494caa7923e28a22) to support credential detection on the newer Nitro hypervisor. This newer hypervisor includes the following instance types: `C5, M5, H1, T3` but possibly others as well.
+
+### NetCDF remote reads
+
+Note that opening remote NetCDF files, such as is done with the GeoTrellis GDALRasterSource requires that the following:
+
+- Linux kernel 4.5+ with `userfaultd` support
+- GDAL 2.4.0+
+
+# RPMs
+
+## Fetching
 
 From this directory, issue `./fetch s3://bucket/prefix/` where
 `s3://bucket/prefix/` is the path to a "directory" on S3 where RPMs have been
 previously published.  This will download the available RPMs as if you had
 built them yourself.  You may then go to the `docker` directory off the
 project root and follow the instructions for building the image.
```

Comment:
In the interest of the crystal clarity mentioned in previous comment, maybe we shouldn't bury this all the way down here (unless we're providing some redundancy for documentation skimmers).

Reply:
I believe this is at least partially addressed by adding a document relative link to the first section for those that are interested in fetching. Think it should still be better highlighted after the latest changes?